SPECS/containernetworking-plugins/containernetworking-plugins.spec

```diff
@@ -129,8 +119,8 @@ install -p plugins/ipam/dhcp/systemd/cni-dhcp.socket %{buildroot}%{_unitdir}
 %{_unitdir}/cni-dhcp.socket
 
 %changelog
-* Tue Nov 01 2022 Olivia Crain <oliviacrain@microsoft.com> - 1.1.1-3
-- Bump release to rebuild with go 1.18.8
+* Tue Nov 01 2022 Ameya Usgaonkar <ausgaonkar@microsoft.com> - 1.1.1-3
```

Comment:
Nit: changelog seems incorrect.

Reply:
Ah! As part of this PR, the above file has been deleted from EXTENDED specs and moved to SPECS.  Hence, that change could not be captured in my PR.  I'll bring that in.